lab-integration/src/main.ts

```diff
@@ -0,0 +1,42 @@
+import { DateTime } from 'luxon';
+
+import { promises as fs, watch } from "fs";
+import path from "path";
+import { TestsResult } from "./types";
+
+const filePath = path.join(process.cwd(), "../test-results.json");
+
+(async () => {
+  console.info("SCRIPT IS RUNNING...");
+
+  let lastTestResultDate = DateTime.now();
+
+  watch(filePath, async (eventType) => {
+    const stringifiedDataset = await fs.readFile(filePath, "utf8");
+    const parsedDataset = JSON.parse(stringifiedDataset) as TestsResult[];
+    const dataset = parsedDataset.map((testResult) => {
+      const Data = DateTime.fromFormat(testResult.Data, 'dd/LL/yyyy, HH:mm:ss', { locale: 'pt-BR' });
+
+      return {
+        ...testResult,
+      Data: Data.isValid ? Data : DateTime.now()
+    }
+  });
+
+    if (eventType === "change") {
+      const newTestsResults = dataset.filter(
+        (testResult) => {
+          console.info(lastTestResultDate)
+         return testResult.Data.diff(lastTestResultDate).milliseconds > 0;
+        }
+      );
+      if(dataset.length > 0)
+        lastTestResultDate = dataset.sort((a, b) => b.Data.diff(a.Data).milliseconds)[0].Data as unknown as DateTime<true>;
+
```

Comment:
Let's wrap this `if` statement with a curly bracket for consistency. We did that in line 26.